src/chrome/chromeConnection.ts

```diff
@@ -162,16 +162,17 @@ export class ChromeConnection {
     /**
      * Attach the websocket to the first available tab in the chrome instance with the given remote debugging port number.
      */
-    public attach(port: number, url?: string): Promise<void> {
+    public attach(port: number, url?: string, address?: string): Promise<void> {
```

Comment:
I thought about it. But since port is required, the other two are optional.
Having port, address, url looks equally weird as well.

On Tuesday, June 21, 2016, Kenneth Auchenberg <notifications@github.com>
wrote:

> In src/chrome/chromeConnection.ts
> <https://github.com/Microsoft/vscode-chrome-debug-core/pull/50#discussion_r67997645>
> :
>
> > @@ -162,16 +162,17 @@ export class ChromeConnection {
> >      /**
> >       * Attach the websocket to the first available tab in the chrome instance with the given remote debugging port number.
> >       */
> > -    public attach(port: number, url?: string): Promise<void> {
> > +    public attach(port: number, url?: string, address?: string): Promise<void> {
>
> Logically I'd expect tha arguments to be : Address, port and url.
>
> This smells of an after-thought... just one more param, right? I'd rather
> make sure the arguments are ordered logically, bump the sem-major version
> and update the two consumers of this debugger (edge + chrome)
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/Microsoft/vscode-chrome-debug-core/pull/50/files/813face911502dab6c1b5d5bb3b3c8f2e2a7b9c9#r67997645>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AA-JVFpWtl7_wmkZg5pp2ywh03JzgMlGks5qOMzrgaJpZM4I7Oc5>
> .
>